src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -660,7 +660,12 @@ private void finalizeBuildingSectionPanel(
                     buildTeamsStatisticsTableForSectionPanel(sectionPanel, responsesGroupedByTeam, 
                                                              teamsWithResponses);
                 }
-                sectionPanel.setDisplayingTeamStatistics(true);
+                
+                Map<String, Boolean> isTeamDisplayingStatistics = new HashMap<>();
+                for (String team : teamsWithResponses) {
+                    isTeamDisplayingStatistics.put(team, bundle.rosterTeamNameMembersTable.containsKey(team));
+                }
+                sectionPanel.setDisplayingTeamStatistics(isTeamDisplayingStatistics);
```

Comment:
Is it possible to have some teams who are anonymous and others who aren't in `teamWithResponses`?